src/refresh-indicator.jsx

```diff
@@ -24,7 +24,7 @@ const RefreshIndicator = React.createClass({
 
     /**
      * Override the theme's color of the indicator while
-     * it's status is "loading" or it's percentage is 100.
+     * it's status is "loading".
```

Comment:
I haven't tested it. I was just looking at the source code. I may have overlook somethinh.